bitnami/redis/README.md

```diff
@@ -138,35 +140,39 @@ The following table lists the configurable parameters of the Redis chart and the
 | `master.persistence.size`                     | Size of data volume                                                                                                                                 | `8Gi`                                                   |
 | `master.persistence.matchLabels`              | matchLabels persistent volume selector                                                                                                              | `{}`                                                    |
 | `master.persistence.matchExpressions`         | matchExpressions persistent volume selector                                                                                                         | `{}`                                                    |
+<<<<<<< HEAD
```

Comment:
@carrodher @jotamartos I think you broke something here ;)

Reply:
Fixed!